scripts/build/cht-core.yml.template

```diff
@@ -41,6 +41,35 @@ services:
     networks:
       - {{ network }}
 
+  cht-nginx:
+    image: {{{ repo }}}/cht-nginx:{{ tag }}
+    container_name: {{ nginx_container_name }}
+    depends_on:
+      - cht-api
+      - haproxy
+    ports:
+      - "${NGINX_HTTP_PORT:-80}:80"
```

Comment:
I think that, right now, API also exposes its port (5988) and all of our e2e tests pass requests directly to API instead of nginx. Can we run e2e tests and only connect to nginx?

Reply:
I think so. If we do that we should also remove direct access to API (`expose` not `ports`). Do you think it's worth doing in this PR?

Reply:
Not necessarily in this PR, but I think in the final version, we should run all tests through nginx.

Reply:
Ok. If you're happy merging this as is, I'll get started on the branch to run e2e tests through nginx and block direct API access.